packages/assets-controllers/src/TokenListController.test.ts

```diff
@@ -638,16 +608,11 @@ describe('TokenListController', () => {
       .persist();
 
     const controllerMessenger = getControllerMessenger();
-    const { network } = setupNetworkController(controllerMessenger);
     const messenger = getRestrictedMessenger(controllerMessenger);
-
+    let onNetworkStateChangeCallback: any;
```

Comment:
What do you think about providing the type here instead of using `any`?

Reply:
When i add the type: `let onNetworkStateChangeCallback: (state: NetworkState | ProviderConfig) => void;`

I get an error when trying to use it:
```
FAIL assets-controllers src/TokenListController.test.ts
➤ YN0000: [@metamask/assets-controllers]:   ● Test suite failed to run
➤ YN0000: [@metamask/assets-controllers]: 
➤ YN0000: [@metamask/assets-controllers]:     src/TokenListController.test.ts:625:5 - error TS2454: Variable 'onNetworkStateChangeCallback' is used before being assigned.
➤ YN0000: [@metamask/assets-controllers]: 
➤ YN0000: [@metamask/assets-controllers]:     625     onNetworkStateChangeCallback({ chainId: '5' });
````

Reply:
resolved by changing to: 

`    let onNetworkStateChangeCallback!: (state: NetworkState | ProviderConfig) => void;`

Reply:
Ah, nice, and I see why you had to add the `!` here. Is this documented anywhere in the TypeScript documentation to your knowledge? The only reference I could really find (at least for this particular usage) was here: https://devblogs.microsoft.com/typescript/announcing-typescript-2-7/#definite-assignment-assertions